schema/postgresql/get_sla_ok_percent.sql

```diff
@@ -127,12 +139,12 @@ after AS (
 	,tstzrange(state_time, COALESCE(lead(state_time) OVER w, endtime),'(]') AS zeitraum
 		--,lead(state_time) OVER w - state_time AS dauer
 	FROM (
-		SELECT state > 1 AS down
+		SELECT state > crit.value AS down
```

Comment:
quick question: when I looked at the code I wondered whether the `OVER w > 1` needed to be changed to `OVER w > 0` for the host case as well, can you confirm that this is not needed?

Reply:
No, can't confirm. But it checks for the proper condition now ;-)